src/sentry/incidents/logic.py

```diff
@@ -1221,6 +1222,13 @@ def create_alert_rule_trigger_action(
             trigger.alert_rule.organization, sentry_app_id, installations
         )
 
+    # store priority in the json sentry_app_config
+    if priority is not None and type in [ActionService.PAGERDUTY, ActionService.OPSGENIE]:
+        if sentry_app_config:
+            sentry_app_config.update({"priority": priority})
```

Comment:
what is the key already exists, do we want to override?

Reply:
if you explicitly pass in the priority, then i think we should override

Reply:
hmmm, then should we just do this at the `sentry_app_config` level where it's being created?

Reply:
this reason i did this was because i didn't want to create a new field in the model just to store the priority. and i added the ability to pass in the `priority` directly into the API which seems more obvious to me than passing it within `sentry_app_config` which is not aptly named at all